spec/views/generic_file/_browse_everything.html.erb_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+describe 'generic_files/_browse_everything.html.erb', :type => :view do
+  it 'shows user timing warning' do
+    render
+    page = Capybara::Node::Simple.new(rendered)
+    expect(page).to have_selector('div.alert-success', text: /Uploading from cloud providers is time sensitive/i , count: 1)
```

Comment:
Change text to "Please note that if you upload a large number of files" per above?